components/image-builder-bob/pkg/proxy/proxy.go

```diff
@@ -52,7 +52,7 @@ type Repo struct {
 	Auth func() docker.Authorizer
 }
 
-func rewriteURL(u *url.URL, fromRepo, toRepo, host, tag string) {
+func rewriteDockerAPIURL(u *url.URL, fromRepo, toRepo, host, tag string) {
```

Comment:
can you add unit tests for this please. just to be safe.

Reply:
Done